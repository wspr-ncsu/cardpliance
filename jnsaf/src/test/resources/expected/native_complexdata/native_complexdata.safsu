`Lorg/arguslab/native_complexdata/MainActivity;.send:(Lorg/arguslab/native_complexdata/ComplexData;)V`:
;