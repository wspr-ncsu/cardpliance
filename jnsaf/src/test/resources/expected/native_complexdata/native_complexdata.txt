Lorg/arguslab/native_complexdata/MainActivity;.send:(Lorg/arguslab/native_complexdata/ComplexData;)V -> _SINK_ 1